packages/react-components/react-spinbutton/src/components/SpinButton/SpinButton.test.tsx

```diff
@@ -148,34 +148,30 @@ describe('SpinButton', () => {
     expect(getSpinButtonInput().value).toEqual('100');
   });
 
-  it('does not clamp the value when outside the bounds when uncontrolled', () => {
+  it('clamps the value when outside the bounds when uncontrolled', () => {
     const onChange = jest.fn();
     const { getAllByRole } = render(<SpinButton defaultValue={100} min={0} max={10} onChange={onChange} />);
 
-    const [incrementButton, decrementButton] = getAllByRole('button');
+    const [incrementButton] = getAllByRole('button');
```

Comment:
You could maybe add an `expect` assertion before the click to verify that the value before it is not clamped.